authz/authz-cr-validation/run-test.sh

```diff
@@ -45,3 +45,14 @@ check_cr_validation "./authz/authz-cr-validation/invalid_provider_deny.yaml" "tr
 check_cr_validation "./authz/authz-cr-validation/invalid_provider_custom.yaml" "true"
 check_cr_validation "./authz/authz-cr-validation/invalid_multiple_providers.yaml" "true"
 check_cr_validation "./authz/authz-cr-validation/valid.yaml" "false"
+
+check_cr_validation "./authz/authz-cr-validation/backend_ref_invalid_group.yaml" "true"
```

Comment:
can you add it in a new shell script. Let's not merge authzExtension and authzPolicy.

Reply:
Done.